Setup.py

```diff
@@ -90,12 +94,19 @@ def check_profile(profile):
 
 
 def get_gcc_version():
```

Comment:
Maybe we should rename this function to something like `get_compiler_version`

Reply:
Done